func/python/py_func.cpp

```diff
@@ -161,13 +164,21 @@ int main(int argc, char* argv[])
 
             // Check return value and clear up
             if (returnValue != nullptr) {
-                printf("Python call succeeded (return value = %ld)\n",
-                       PyLong_AsLong(returnValue));
+                long r = PyLong_AsLong(returnValue);
+
+                if (r == 0) {
+                    printf("Python call succeeded\n");
+                } else {
+                    printf("Python call failed (return value = %ld)\n", r);
+                }
```

Comment:
This now more strictly enforces Python functions returning zero to be successful (rather than just returning anything)